src/backend/templates/tracks.hbs

```diff
@@ -836,9 +853,27 @@
             if ($('.date-filter:visible').length == 0  && searchTracks==1) {
                 $('#no-results').remove();
                 $('.date-list').after("<p style = 'padding-left: 15px; margin-top: 15px; font-weight: bold; color: red ' id='no-results'>No matching results found.</p>");
+
+                if(trackFilterMode){
+                  $('.track-filter-info').addClass('filter-not-found');
+                  $('.track-filter-info').css("border","1px dotted red")
+
+                } else if(roomFilterMode) {
+                    $('.room-filter-info').addClass('filter-not-found');
+                    $('.room-filter-info').css("border","1px dotted red")
+                }
+
             } else {
                 $('#no-results').remove();
 
+                if(trackFilterMode){
+                  $('.track-filter-info').removeClass('filter-not-found');
+                  $('.track-filter-info').css("border","1px dotted black")
```

Comment:
@shubham3211 store ` $('.track-filter-info')` in a variable and use it everywhere.